pkgs/applications/editors/vscode/generic.nix

```diff
@@ -108,6 +106,10 @@ let
 
     preFixup = ''
       gappsWrapperArgs+=(
+        --set GTK_USE_PORTAL 1
+        ${ # we cannot use runtimeDependencies otherwise libdbusmenu do not work on kde
```

Comment:
Can you link the ticket where it's explained why this is the case?

Reply:
```suggestion
        # we cannot use runtimeDependencies, otherwise libdbusmenu does not work on kde,
        # see https://github.com/NixOS/nixpkgs/issues/65680
        ${
```